oneanddone/base/templates/base/base.html

```diff
@@ -129,7 +129,7 @@
           <ul class="footer-nav">
             <li><a href="http://twitter.com/firefox">Twitter</a></li>
             <li><a href="http://facebook.com/Firefox">Facebook</a></li>
-            <li><a href="https://affiliates.mozilla.org/">{{ _('Firefox Affiliates') }}</a></li>
+            <li><a href="https://friends.mozilla.org/">{{ _('Firefox Friend) }}</a></li>
```

Comment:
Thanks for the pull request, @ghost. This looks good, but I think the link should read `Firefox Friends` not `Firefox Friend`. Would you mind fixing that?